src/lib/concurrency/transaction_context.hpp

```diff
@@ -60,10 +60,10 @@ class TransactionContext : public std::enable_shared_from_this<TransactionContex
   TransactionID transaction_id() const;
 
   /**
-   * The last commit id represents the snapshot in time of the database
+   * The as-of commit id represents the snapshot in time of the database
    * that the transaction is able to see and access.
    */
-  CommitID last_commit_id() const;
+  CommitID as_of_commit_id() const;
 
   /**
    * Only available after TransactionManager::prepare_commit has been called
```

Comment:
This only describes when `commit_id` is available, not what it actually means. Now with `as_of_commit_id` it took me a minute to realize the difference. Maybe update this comment as well?

Reply:
maybe that wasn't the best LoC to comment on. This is for `TransactionContext::commit_id`

Reply:
Good point. Added.